components/Chips/src/MDCChipView.h

```diff
@@ -151,7 +151,18 @@
 
  Default value is YES.
  */
-@property(nonatomic, readwrite, setter=mdc_setLegacyFontScaling:) BOOL mdc_legacyFontScaling;
+@property(nonatomic, readwrite, setter=mdc_setLegacyFontScaling:)
+    BOOL mdc_legacyFontScaling __deprecated;
+
+/**
+ Enable font scaling curves fallback behavior for Dynamic Type.
```

Comment:
Suggested docs:

```
Affects the fallback behavior for when a scaled font is not provided.

If enabled, the font size will adjust even if a scaled font has not been provided for
a given UIFont property on this component.

If disabled, the font size will only be adjusted if a scaled font has been provided. This behavior
most closely matches UIKit's.

Default value is YES, but this flag will eventually default to NO and then deleted.
```

Reply:
Thanks for the suggestion. Addressed in 38efb39b99ee4b07d1fea3a0e0592ad49556a126.